src/main/resources/view/LogCard.fxml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.VBox?>
+<HBox xmlns:fx="http://javafx.com/fxml/1" id="cardPane" fx:id="cardPane"
+      xmlns="http://javafx.com/javafx/17">
+    <GridPane HBox.hgrow="ALWAYS">
+        <columnConstraints>
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150"/>
+        </columnConstraints>
+        <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+            <padding>
+                <Insets top="5" right="5" bottom="5" left="15"/>
+            </padding>
+            <HBox spacing="0.5" alignment="CENTER_LEFT">
+                <Label fx:id="id" styleClass="cell_big_label">
+                    <minWidth>
+                        <!-- Ensures that the label text is never truncated -->
+                        <Region fx:constant="USE_PREF_SIZE"/>
+                    </minWidth>
+                </Label>
+                <Label fx:id="date" text="\$date" styleClass="cell_big_label"/>
+            </HBox>
+            <Label fx:id="remark" styleClass="cell_small_label" text="\$remark"/>
```

Comment:
Right now if we have a really long remark, it'll get cut off and there's no way of reading it. 
Simple solution could be to add wrapText="true" to the <Label> element? 
It would look a bit ugly but it'll be functional hahahhaha

<img width="1021" alt="Screenshot 2024-10-24 at 10 05 30 AM" src="https://github.com/user-attachments/assets/7d73956c-b6cd-49b4-b622-2ab29b895c20">